roachtest: run Read Committed variants of TPC-C without txn retry loops

nvb opened the pull request:
Closes #115191.
Depends on #61986.

This commit switches the two nightly Read Committed variants of the TPC-C roachtest to run without transaction retry loops, using the `--txn-retries` flag introduced in #117096. With #117630 and #61986 resolved (the latter of which is still in review and under development), these tests both pass.

Release note: None

nvb committed:
roachtest: run Read Committed variants of TPC-C without txn retry loops

Closes #115191.
Depends on #61986.

This commit switches the two nightly Read Committed variants of the TPC-C
roachtest to run without transaction retry loops, using the `--txn-retries` flag
introduced in #117096. With #117630 and #61986 resolved (the latter of which
is still in review and under development), these tests both pass.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/118000)
<!-- Reviewable:end -->

arulajmani reviewed (approved):
As exciting as a 2 line change gets! :fire: 

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/118000#-:-NoYulZJCSX6jqb3G_yU:bl53yzv)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @DarrylWong, @michae2, and @srosenberg)



<!-- Sent from Reviewable.io -->

michae2 reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> Here we go! *rubs hands together*

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/118000#-:-NoZ0VRkD27638diyT8k:b-z33vae)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @DarrylWong and @srosenberg)



<!-- Sent from Reviewable.io -->

nvb committed:
roachtest: increase vmodule for RC variants of TPC-C

This commit increases the vmodule level around transaction
pushes so that if we do see a transaction retry error, we
can debug it.

Epic: None
Release note: None

nvanbenschoten commented:
bors r+
